src/issue-filing/github/github-url-rectifier.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export type UrlRectifier = (url: string) => string;
+
+export const rectify: UrlRectifier = (url: string): string => {
+    const ownerAndRepo = new RegExp('^https?://github.com/[^/]+/[^/]+$');
+    if (ownerAndRepo.test(url)) {
+        return `${url}/issues`;
+    }
+
+    const ownerAndRepoAndSlash = new RegExp('^https?://github.com/[^/]+/[^/]+/?$');
```

Comment:
The last ? here is not correct because this RegExp is looking for a URL that ends with a slash
```suggestion
    const ownerAndRepoAndSlash = new RegExp('^https?://github.com/[^/]+/[^/]+/$');
```
(Doesn't cause a bug because the case without a slash is handled by lines 6-9 above)

Reply:
Good catch